kubernetes/zenko/values.yaml

```diff
@@ -117,17 +114,42 @@ zenko-queue:
     enabled: true
   configurationOverrides:
     "offsets.topic.replication.factor": 3 # - replication factor for the offsets topic
-    "auto.create.topics.enable": true     # - enable auto creation of topic on the server
+    "auto.create.topics.enable": false    # - enable auto creation of topic on the server
     "min.insync.replicas": 2              # - min number of replicas that must acknowledge a write
-  partitionCount: *nodeCount
-  partitionFactor: *crrParallelism
+    "message.max.bytes": "5000000"        # - the largest record batch size allowed
   prometheus:
     jmx:
       enabled: true
     kafka:
       enabled: true
-zookeeper:
-  enabled: false # Disables zenko-queue from deploying its own quorum
+  topics:
+    - name: backbeat-gc
+      partitions: *nodeCount
+      replicationFactor: 3
+    - name: backbeat-ingestion
+      partitions: *nodeCount
+      replicationFactor: 3
+    - name: backbeat-lifecycle-object-tasks
+      partitions: *nodeCount
+      replicationFactor: 3
+    - name: backbeat-lifecycle-bucket-tasks
+      partitions: *nodeCount
+      replicationFactor: 3
+    - name: backbeat-metrics
+      partitions: 1
```

Comment:
why not `*nodeCount` here?

Reply:
@jonathan-gramain `BackbeatAPI` does not depend on `nodeCount` since it's preferred for it to only have 1 instance. So it is like this to follow the `"to have one partition per running process"` statement.

Reply:
Unless some other process uses this topic besides API?